scripts/globals/player.lua

```diff
@@ -134,6 +134,7 @@ local function CharCreate(player)
     player:addItem(536) -- adventurer coupon
     player:addTitle(dsp.title.NEW_ADVENTURER)
     player:setVar("MoghouseExplication", 1) -- needs Moghouse intro
+	player:setVar("FishingCatches", 0) -- counter for fishing
```

Comment:
You've placed this is a section that only applies to newly created characters, at which point this variable would already be zero (any variable not found returns zero)

Reply:
Done